docker/worker/Dockerfile_x86

```diff
@@ -0,0 +1,31 @@
+#!BuildTag: openqa_worker_x86
+FROM opensuse/leap:15.1
```

Comment:
Why 15.1 on x86_64 and not 15.2? 🤔

Reply:
I've been asking myself the same question.